app.py

```diff
@@ -169,15 +169,21 @@ def before_request_https():
 
 # Unique random key for sessions.
 # For settings with multiple workers, an environment variable is required, otherwise cookies will be constantly removed and re-set by different workers.
-app.config['SECRET_KEY'] = os.getenv ('SECRET_KEY') or uuid.uuid4().hex
+if utils.is_production():
```

Comment:
As long as we have a single machine on alpha or test, we don't need this check, so I'm ok with the change.